package/gocryptfs/package

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(gocryptfs)
+pkgdesc="An encrypted overlay filesystem written in Go."
+url="https://nuetzlich.net/gocryptfs/"
+_srcver=2.0-beta2
+pkgver="$_srcver"-1
+timestamp=2021-03-22
+section=utils
+maintainer="plan5 <30434574+plan5@users.noreply.github.com>"
+license=MIT
+
+makedepends=()
+installdepends=(fuse-utils)
```

Comment:
This will need to be updated to require the fuse module on rM1. @matteodelabre do we have a mechanism to have different install dependencies between reMarkable versions?

Reply:
We could depend on a metapackage like we do for `display`.

Reply:
@danshick could you add information to the fuse PR with how to accomplish that for @plan5?

Reply:
`installdepends_rm1=(fuse-utils)` will make the dependency rM1-only

Reply:
@matteodelabre Which is better, two copies of the package or a metapackage?

Reply:
To be precise, the `display` solution is not technically using a metapackage: the `display` package exists, is empty on rM1, and contains rm2fb on rM2. You could do that also for the fuse module (i.e., make it contain the module on rM1 and be empty on rM2). I think that would preferable since it would save space on our server compared to duplicating the gocryptfs package (I that’s about the only meaningful difference between the two approaches).

Reply:
You're right. Was just looking at it now. I think that's the right approach.

Reply:
Added a comment over in the fuse package pointing to the relevant parts of `display`.

Reply:
I've removed the fuse archs section entirely and added a logic to the package section to only install files on rm1. 
The install dependency in gocryptfs is now only pulled for rm1 via installdepends_rm1.
Does this resolve it all?

Reply:
You can actually now make the fuse package a dependency for all architectures. Because the package is empty for rm2, it'll "install" it, but the package is basically a noop.

This means that there's only one gocryptfs package instead of two, which is desirable because the fuse package is actually the one that needs to vary by device.

Reply:
Ok, I've moved it to the general installdepends.

btw I will add the gocryptfs simple-script GUI as a separate package soon.
I'll have to first find an elegant way to launch it before launchers/xochitl via systemd.
I've already asked about this in the Discord/Matrix-Room but if you have a suggestion I'm all ears :)

Reply:
I think systemd allows units to specify that they need to launch before something else does.